ofl/reemkufifun/METADATA.pb

```diff
@@ -17,13 +17,12 @@ subsets: "latin"
 subsets: "latin-ext"
 subsets: "menu"
 subsets: "vietnamese"
-primary_script: "Arab"
```

Comment:
`gftools add-font` is stripping this out so I'll readd and fix gftools.

Reply:
Will be fixed once https://github.com/googlefonts/gftools/pull/606 is merged and a new release pushed.